Traceback (most recent call last):
  File "g:\_2023\courses\Sweigart\python-practice\boring_stuff\11_debugging\about_traceback.py", line 4, in <module>
    raise Exception("This is the error message.")
Exception: This is the error message.
